### ISSUE TITLE
WCAG ISSUE - the left and right icons on the slider do not meet the Web Content Accessibility Standard

### ISSUE THREAD

**Issue by iqbalhood** [topcoder-archive/beta-testing-dashboard#368]
On the main slider, the left and right icons do not meet the Web Content Accessibility Standard for some images content.
maybe the left and right elements should be replaced with ones that meet the WCAG standard

<img width="786" alt="Screen Shot 2021-09-16 at 00 31 13" src="https://user-images.githubusercontent.com/1563756/133481662-027cd6be-3e1a-4a82-8b43-2654b48410fd.png">

**Comment by kkartunov** [comment]
Not true bug in sense of broken functionality.
